components/analytics/compact-dashboard.tsx

```
@@ -0,0 +1,1298 @@
+                                                        const centerClass = (needsCentering && isLastItem) ? "col-span-2 mx-auto max-w-[160px]" : "";
+
+                                                        return (
+                                                            <div
+                                                                key={`platform-${platform}`}
+                                                                className={`flex flex-col items-center justify-center ${centerClass}`}
+                                                                onClick={() => selectBotsByCategory(platformBots)}
+                                                                onMouseEnter={() => setHoveredBots(platformBots)}
+                                                                onMouseLeave={() => setHoveredBots([])}
+                                                                style={{ cursor: 'pointer' }}
+                                                            >
+                                                                <div className="text-sm font-medium mb-2 capitalize">{platform}</div>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)